tests/tecc

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -e
+# Copyright (C) 2022 Simo Sorce <simo@redhat.com>
```

Comment:
nit: 2023 ;)

Reply:
I didn't change contents, just move around stuff :)

Reply:
ok, missed that it was just moved around from the eccsha2. Thanks for clarification